Orange/data/io.py

```diff
@@ -369,7 +369,8 @@ def _equal_length(lst):
 
             if type_flag in StringVariable.TYPE_HEADERS:
                 coltype = StringVariable
-
+                values = [None if isinstance(i, float)
```

Comment:
@VesnaT, I was also thinking about empty strings as an alternative to `None`. Empty strings are also false, and they have the same type, so it may be more practical.

Also, consider the attribute contains gene names. Empty name is then the same as missing name. It is probably uncommon to have a "known, but empty" value.